guides/deploying-to-digital-ocean-with-ansible.md

```diff
@@ -0,0 +1,167 @@
+
+Open Commerce services will be exposed to the public using [Traefik](https://github.com/containous/traefik/), which 
+is a cloud native router. Traefik will act as a reverse proxy that will route traffic to Docker containers. As stated above, you will need a registered domain to complete this step, as it will be necessary to manage DNS records for it.
+
+This guide will use the following subdomains, where `example.com` will need to substitute it with your domain:
+
+| Subdomain              | Description                       |
+| ---------------------- |-----------------------------------|
+| api.example.com        | The open Commerce GraphQL API     |
+| storefront.example.com | The example storefront            |
+| admin.example.com      | The Open Commerce admin interface |
+| traefik.example.com    | Traefik's admin UI                |
+
+Each of your domains will need an `A` DNS record that resolves to your host's IP. It's recommend to use DigitalOcean's free [DNS manager](https://www.digitalocean.com/docs/networking/dns/overview/). Further, in order to obtain SSL certificates for your subdomains, you will need a DNS manager that supports [CAA](https://support.dnsimple.com/articles/caa-record/) records.
+
+
+Further, you will need a [DigitalOcean Auth token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) to generate CAA records for your sub-domains.
+
+## Clone the Proxy-Traefik Repo to your local machine
+
+The project that contains all the playbooks, etc. is located [here](https://github.com/reactioncommerce/proxy-traefik).
+
+In order to use these files you need to have a local copy on your machine. To clone the files locally just run:
+```sh
+git clone git@github.com:reactioncommerce/proxy-traefik.git
+```
+
+To make the project disconnected from the original repository you should `rm -rf .git` from within the newly created directory and then `git init`.
+You should not push any of your changes, especially your DO key, to a public repository and there is no need to do so since the command are intended to run from your local machine.
+
+## Prepare the Remote Host
+
+In this guide a DigitalOcean node will be used to host the Open Commerce Platform. If you don't yet have an account, 
+create one at [digitalocean.com](https://digitalocean.com). Once you are signed in to your account, create a new droplet using the Ubuntu 18.4 image with at least 2GB of RAM. Enable DigitalOcean's [free firewall](https://www.digitalocean.com/docs/networking/firewalls/) and add inbound rules for SSH, HTTP, HTTPS and add your droplet to the firewall.
+
+After the droplet is created either select an existing SSH key to login or click on the "New SSH Key" under the authentication section  and copy your public SSH key from your local computer.
+
+Copy the newly created IP address and verify that you can log in into the new server by executing:
```

Comment:
Copy the newly created IP address and verify that you can log ~~in~~ into the new server by executing: